pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -382,7 +385,10 @@ func (r *ReconcileCredentialsRequest) Reconcile(ctx context.Context, request rec
 		return reconcile.Result{}, fmt.Errorf("configuration conflict")
 	} else if mode == operatorv1.CloudCredentialsModeManual {
 		logger.Infof("operator set to disabled / manual mode")
-		return reconcile.Result{}, err
+		if stsDetected {
+			logger.Infof("operator detects STS enabled cluster")
```

Comment:
We now log this and the message above for each CR reconcile which is slightly contradictory. Does it make sense to combine the spirit of the message in some way?

```
time="2023-06-23T14:07:13Z" level=info msg="operator set to disabled / manual mode" controller=credreq cr=openshift-cloud-credential-operator/manila-csi-drive
r-operator
time="2023-06-23T14:07:13Z" level=info msg="operator detects STS enabled cluster" controller=credreq cr=openshift-cloud-credential-operator/manila-csi-driver-
operator
```

Reply:
this is take care of now seemingly